src/interactive/widgets/mark.rs

```diff
@@ -374,9 +374,10 @@ impl MarkPane {
                 #[cfg(feature = "trash-move")]
                 Span::styled(" to trash or ", default_style),
                 Span::styled(
-                    " Ctrl + r",
+                    " Ctrl + r ",
                     Style {
                         fg: Color::LightRed.into(),
+                        bg: Color::Black.into(),
```

Comment:
I copied this line from above, but it may conflict with the `add_modifier` style below? I don't really know what that means or how to trigger it.